### ISSUE TITLE
Griffith Uni 2017 Trimester 2 Undergrad Projects

### ISSUE THREAD

**Issue by mcoster** [OpenSourceMalaria/OSM_To_Do_List#508]
Six students undertaking undergraduate projects at Griffith University under my supervision will target series 4 molecules of interest this trimester ~ July - September. Below is a scheme showing what we propose for them to work on. They will start next week and have a bunch of OH&S paperwork to complete, etc. One of their first tasks is also to propose a synthesis of their target molecule. We will post again with our proposed routes, once they have had a chance to do the planning. Any and all comments and suggestions welcome!

![osm_series4_synthetic_targets-deidentified](https://user-images.githubusercontent.com/28282596/27619775-3d6fd742-5c08-11e7-9fb0-6ee158467123.gif)

[OSM_Series4_synthetic_targets-deidentified.zip](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/1107511/OSM_Series4_synthetic_targets-deidentified.zip)

**Comment by MFernflower** [comment]
@mattodd Wasn't the carboxylic acid compound successfully made? @mcoster Perhaps for student LK they could make the alkene instead?

**Comment by mattodd** [comment]
Progress to hERG evador (or the "acid test") is, I think, [here](http://www.openwetware.org/wiki/OpenSourceMalaria:Triazolopyrazine_(TP)_Series#Desirable_Compounds_Not_Yet_Synthesised), unless there has been progress since then by any of @edwintse @rbroadrup-HC @ghost ? #451

**Comment by mcoster** [comment]
Yep, as best I can tell, following all relevant posts, the hERG evador hasn't been successfully synthesised.

**Comment by OHaraJansonSchankler** [comment]
Hello all, I am going into my final year at Haverford College working in Professor Broadrup's lab and the hERG evador is one of the targets toward which I have been working. NMR and GC-MS spectra (tentatively) taken this week show that I have achieved Alice Williamson's product for AEW 287:

<img width="483" alt="2017-06-30 17 50 42" src="https://user-images.githubusercontent.com/21692047/27755249-95204dc2-5dbc-11e7-8b26-aaa5af5ca9ae.png">



To give a quick update, I have just dried the worked up product of the reaction which has AEW 289 precedent:

<img width="446" alt="2017-06-30 17 52 34" src="https://user-images.githubusercontent.com/21692047/27755298-d2b8a90e-5dbc-11e7-81e7-4e0c65bffde7.png">

I will not be able to get into lab to purify the dried crude until next week and I will post spectra on this thread after I take time this weekend to transfer my lab notebook to the digital project lab notebooks page.

Best,
George O'Hara (Haverford College Class of 2018)

**Comment by mattodd** [comment]
Thanks @OHaraJansonSchankler - yes we really need to see the data you obtained with @rbroadrup-HC , and I'm sure this will help @mcoster with planning for his class. Great if you've made progress towards this compound.

**Comment by mcoster** [comment]
Thanks for the update @OHaraJansonSchankler! Looks like we would be best to revise LK's target, rather than duplicate activity. The GU team are meeting on Wednesday. Perhaps LK could look at bromination/chlorination of "Jasper 2" or "Jasper 3" - #493 

Or is @jaspert95 still working on this?

@mattodd

**Comment by mattodd** [comment]
No, @jaspert95 just finished. He'll post his full report ASAP, but yes, this halogenation still needs completing, as do all the so-called "Norcross" compounds we could access. This is something that might be of interest to a new PhD student here in Sydney, @maratsydney , who is just getting started, and it might be something that an incoming exchange student from Nanjing is interested in doing too. If we coordinate activities here we could really nail this route, I think, and there's lots to do. If you wanted to pursue the hERG evador though, I'm sure there's something to be done there - I sense that even if we get the THP off that alcohol, oxidation to the acid (and then handling of the final compound) will present significant challenges. I'm not sure whether @OHaraJansonSchankler is implying that the final compound has been made, or whether it's an intermediate.

**Comment by OHaraJansonSchankler** [comment]
Hello, still transferring lab notebook information--I should clarify that the compound I have achieved in the hERG pathway is an intermediate and not the final product. Further to this, from the material I have now, assuming the latest step worked (AEW 289 precedent), as you say @mattodd the alcohol needs to be de-protected. I have also been considering different ways to oxidize the resulting alcohol to a carboxylic acid, given that part of the compound was reduced from acid to alcohol in a previous step, if anyone has suggestions as to how to achieve this oxidation.

**Comment by mcoster** [comment]
I don't have a good 'feel' for the chemistry of these triazolopyrazines, eg. how sensitive are they to oxidants? For sensitive molecules, we've had much better results with two step procedure, ie. Swern/Dess-Martin to aldehyde, then Pinnick oxidation to carboxylic acid,  than any direct oxidation.

**Comment by mcoster** [comment]
Planning continues at GU. The students have been tasked with getting to grips with their target molecule and planning out its synthesis. We will meet tomorrow to discuss their strategies, they will do their safety induction on Monday and hopefully they will do their first reaction on Tuesday!

We are awaiting an order of 4-(difluoromethoxy)benzaldehyde - the majority of [DC-NYS](http://www.openwetware.org/wiki/OpenSourceMalaria:Triazolopyrazine_(TP)_Series#Desirable_Compounds_Not_Yet_Synthesised) compounds have the 4-CHF2OAr triazole side-chain. While we wait for the order to come in, the students may wish to practice with some related/model chemistry. We have access to a good library of other benzaldehydes - are there any particular ones that would lead to useful congeners to the [DC-NYS](http://www.openwetware.org/wiki/OpenSourceMalaria:Triazolopyrazine_(TP)_Series#Desirable_Compounds_Not_Yet_Synthesised) molecules if we progressed them to target compounds? I note that *4-cyano* and *4-chloro* turn up in potent compounds quite a bit. We also have some amount of 4-(*trifluoro*methoxy)benzaldehyde.

**Comment by MFernflower** [comment]
have any n-formylmorpholine on hand? it's never been tested before @mattodd

**Comment by mcoster** [comment]
@MFernflower thanks for the suggestion. We don't, although I suspect *N*-formamides probably wouldn't undergo the same sequence of condensation with pyrazinohydrazide/PIDA cyclisation as benzaldehydes do. Anyone else care to comment? Looks like @edwintse has been [working on piperidinyl substituted compounds](https://au-mynotebook.labarchives.com/share/Edwin%20Tse/NjYzLjB8MTUwOS81MTAvVHJlZU5vZGUvNDQ0Mzk2MjExfDE2ODMuMA==#!) via cross coupling with the triazole bromide moiety.

**Comment by OHaraJansonSchankler** [comment]
@mcoster Noted, thank you. Will have data on hERG evador synthesis steps up by end of today (US EST)

**Comment by OHaraJansonSchankler** [comment]
Hello, sorry for the delay--product of GO 289 (AEW 289 precedent) was being purified. 

To summarize my progress on the hERG evador, one of the compounds that I have been working to synthesize, from the most recent step attempted to previously attempted steps:
 
GO 289 (AEW 289) precedent:

<img width="442" alt="2017-07-05 0 57 47" src="https://user-images.githubusercontent.com/21692047/27934440-cda4d382-6273-11e7-8c20-074dff89bfa5.png">


The electrophile of this reaction was synthesized here:
 [http://malaria.ourexperiment.org/reduced_alkyl/14592/092216_Synthesis_of_5chloro34difluoromethoxyphenyl124triazolo43apyrazine_rxn_31.html]

289 did not go--proton NMR shows the nucleophile was destroyed, as the THP peaks from the nucleophile disappear when comparing the starting material spectrum to the AEW 287 product that served as the nucleophile, and in turn comparing these two to the experimental product spectrum.

Here is the proton NMR for GO (George O'Hara) 287, with AEW 287 precedent.

<img width="1040" alt="2017-07-06 17 35 48" src="https://user-images.githubusercontent.com/21692047/27934515-2c57330c-6274-11e7-8feb-bf00cf7a08f5.png">

This supports the notion that GO 287 was successful. To compare, here is the proton NMR spectrum of GO 281 (AEW 281 precedent) product, GO 287 starting material.

<img width="1049" alt="2017-07-06 18 00 00" src="https://user-images.githubusercontent.com/21692047/27934683-d63ebb4c-6274-11e7-8a1a-a6435198c85f.png">

Here also is the proton NMR spectrum of GO 266 product (AEW 266 precedent, starting material for AEW 281):

<img width="1052" alt="2017-07-06 18 03 33" src="https://user-images.githubusercontent.com/21692047/27934793-6e6a6c18-6275-11e7-90de-3c954fc88bf4.png">

Here also is the NMR for tropic acid, racemic mixture, starting material for GO 266:

<img width="1056" alt="2017-07-06 18 04 14" src="https://user-images.githubusercontent.com/21692047/27934823-9c24c554-6275-11e7-84cc-25bff3eb57f4.png">


With regard to GO 289 (AEW 289 precedent):

Here is the proton NMR for GO 289 starting material (product of reaction with AEW 205 precedent):

<img width="858" alt="2017-07-06 17 55 52" src="https://user-images.githubusercontent.com/21692047/27934544-439caa42-6274-11e7-8a92-f651753276dd.png">

Here is the proton NMR for GO 289:

<img width="1049" alt="2017-07-06 17 36 21" src="https://user-images.githubusercontent.com/21692047/27934602-732140e8-6274-11e7-8d2a-7581009ab87f.png">

I apologize for the ethyl acetate peaks--it was most likely due to contamination of the deuterated chloroform that the Broadrup lab was using until yesterday as NMR solvent. 

Moving forward, after pushing more material forward, I will re-attempt 289 by heating at 40 Celsius under reflux, as Dr. Alice Williamson did in her precedent. I attempted the 289 reaction at 0 Celsius to avoid nucleophile destruction that, from my intuition, was a contributing factor to Dr. Williamson's low yield (20%), given the black/dark brown colour of the reaction that Dr. Williamson reported soon after starting the reaction, refluxing ether as the reaction solvent at 40 Celsius.

Considering the volatility of ether, how might one approach this reaction?

Please comment with any suggestions you might have. Thank you!

**Comment by OHaraJansonSchankler** [comment]
Hello all, I am just following up on my last post. I have been continuing work on the hERG evador synthesis by pushing more starting material through. I just carried out reaction with AEW 266 on an 8-gram scale (tropic acid starting material) and will post spectral data as soon as I have them.

Best,

--George O'Hara

**Comment by mattodd** [comment]
Hi @mcoster sorry for the delay. 

You asked about sensitivity to oxidants - not sure. There will be some sensitivity, but we’ve not directly assessed it, or isolated byproducts in quantity from oxidations.

You asked about aldehyde substituents. Difluoromethoxy appears best, yes. CN and Cl have given good results, though we’ve had some solubility issues with both in some cases. Trifluoromethoxy has I think been OK, in a way that [methoxy has not](http://www.openwetware.org/wiki/Image:CH3_vs_CHF2_vs_CF3_Northeast.png).

Excited to see your routes, Mark. Where are your students going to post? And do you have a lab notebook solution for when the chemistry magic happens?

@OHaraJansonSchankler  Thanks for posting the spectra, but I have to emphasize again: **we need the data**. If you contribute to OSM you need to be posting the raw data and details of the reaction so that your work can be seen and verified by others as if they were reading your lab book in the lab. Summaries don’t cut it. Please could you try to ensure that reactions are put into an ELN (labtrove or another) so that others can benefit from all your hard work? Otherwise I worry we're going to lose it.

**Comment by mcoster** [comment]
Thanks for the feedback @mattodd. I'll post more details of the planned routes here soon, just want to give the students ample opportunity to do their own planning.

On a different topic, I want them to use the LabTrove based ELN for their experiments and I'm also interested in supporting my teaching in the near future with video content, so I married the two ideas and just posted my first ever YouTube video - 
[a tutorial on how to use the ELN.](https://youtu.be/X_tREXdHRE4)


_Sent from my LGE Nexus 5X using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by mattodd** [comment]
That's a really good video, Mark. You've made it look simple. It is, but the first post is not. Your video helps a great deal with that and will be useful for everyone. Added to Tech Ops page. 

Were you contemplating other videos? As I mentioned in #507 I'm wondering about whether we should migrate wiki content over to Github rather than use OpenWetWare (which is not being updated) so if you were considering making a how-to video for the wiki, then maybe pause. One thing that could use a video, maybe a little later, is how to add compounds to the Master List, but we can thrash out what that might need once your guys are up and running and they might actually want to add their compounds to the list.

**Comment by mcoster** [comment]
Thanks Mat. Yes, planning a further video or two. I know it took me a while to "get" how Github is being used by OSM, and there seems to be a bit of an energy barrier to students jumping on here, so thought I'd do a quick intro to Github and how OSM is using it. Just 5-10 minutes of the basics. Didn't have any plans for the wiki, and sounds like it's worthwhile holding off if I did. Happy to tackle the master list at a later date when that becomes more important for us.

**Comment by mcoster** [comment]
Closing. [Follow-up available here](https://github.com/OpenSourceMalaria/Series4/issues/15).
